mock_drivers/cpp_mock_drivers/src/parser_main.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2019-2020 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <ros/ros.h>
+#include <cpp_mock_drivers/BagParser.h>
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "bag_parser");
+
+    mock_drivers::BagParser parser_node(argv[1]);
```

Comment:
Consider making file path ros parameter instead of argument so it can be runtime checked.